scipy/signal/signaltools.py

```diff
@@ -467,6 +502,12 @@ def convolve(in1, in2, mode='full'):
     numpy.polymul : performs polynomial multiplication (same operation, but
                     also accepts poly1d objects)
 
+    Notes
+    -----
+    ``method='fft'`` only works for numerical arrays as it relies on
+    scipy.signal.fftconvolve. In certain cases (i.e., arrays of objects or when
+    rounding integers can lose precision), ``method='direct'`` is always used.
```

Comment:
You could mention that `direct` mode eventually calls `correlate` under the hood.